web_transport/sdk/impl/web_transport_owt_client_impl.cc

```diff
@@ -250,5 +258,38 @@ void WebTransportOwtClientImpl::FireEvent(
   }
 }
 
+MessageStatus WebTransportOwtClientImpl::SendOrQueueDatagram(uint8_t* data,
+                                                             size_t length) {
+  auto* allocator = client_->quic_session()
+                        ->connection()
```

Comment:
perform check by:
client_->quic_session()->connection()->connected() first before you go through the sending process.

Reply:
Added DCHECK.